ViewModel/ImpactAreaScenario/Results/PerformanceAEPVM.cs

```diff
@@ -53,31 +49,19 @@ private void LoadData(ImpactAreaScenarioResults iasResult, List<ThresholdComboIt
             }
         }
 
-        private void LoadHistogramData(SystemPerformanceResults performanceResults, Threshold threshold)
+        private void LoadHistogramData(ScenarioResults scenarioResults, int impactAreaID, Threshold threshold)
         {
-            ThreadsafeInlineHistogram histogramOfAEPs = performanceResults.GetAEPHistogram();
+            //ImpactAreaScenarioResults results = scenarioResults.GetResults(impactAreaID);
+            
+            IHistogram histogramOfAEPs = scenarioResults.AEPHistogram(impactAreaID, threshold.ThresholdID);
             int[] binCounts = histogramOfAEPs.BinCounts;
-            double[] binsAsDoubles = binCounts.Select(x => (double)x).ToArray();
+            double[] binsAsDoubles = binCounts.Select(x => (double)x / histogramOfAEPs.SampleSize).ToArray();
```

Comment:
It's not very obvious what's going on here.  The API should be doing this kind of logic, not you, why are we dividing it by the sample size?